util/status.cc

```diff
@@ -43,6 +43,9 @@ static const char* msgs[static_cast<int>(Status::kMaxSubCode)] = {
     "Memory limit reached",                               // kMemoryLimit
     "Space limit reached",                                // kSpaceLimit
     "No such file or directory",                          // kPathNotFound
+    // KMergeOperandsInsufficientCapacity
+    "Insufficient capacity for merge operands",
+    "Manual compaction paused",                           // kManualCompactionPaused
```

Comment:
It feels that it exceeds 80 characters too.

Reply:
fixed